neuralpredictors/layers/readouts/multi_readout.py

```diff
@@ -1,107 +1,147 @@
-class MultiReadout(Readout, ModuleDict):
+import torch
+from .base import *
+from .point_pooled import *
+from .gaussian import *
+from .factorized import *
+from .attention import *
+from .pyramid import *
+
+
+class MultiReadoutBase(torch.nn.ModuleDict):
```

Comment:
I'd find it more intuitive if this base class is also in `base.py`. And then I think we can get rid of `multi_readout.py`, and instead have everything in the `__init__.py` So the classes itself, like FullGaussian2d, as well as the instantiated MultiReadouts. Then everything can be conventiently imported from layers/readouts import *.

Reply:
I don't think this is possible, it would be a circular import.